packages/ui5-middleware-onelogin/src/cookieGetter.ts

```diff
@@ -0,0 +1,113 @@
+;
```

Comment:
what's the purpose of this line?

Reply:
that's just my unstructured approach leaving semicolons laying around when deleting lines. Not sure why it was even there :D